docs/source/build/index.rst

```diff
@@ -293,6 +293,8 @@ Main Parameters
 
 .. note:: If you would like to use HL Ursa cryptography for your build, please install `Rust <https://www.rust-lang.org/tools/install>`_ in addition to other dependencies. Learn more about HL Ursa integration `here <../integrations/index.html#hyperledger-ursa>`_.
 
+  If you want to use HL Burrow integration, do not forget to install `Go <https://golang.org/doc/install>`_. Learn more about `HL Burrow Integration <../integrations/index.html#hyperledger-burrow>`_.
```

Comment:
It is required not only "go", but also the protobuf module, it is described here:
https://developers.google.com/protocol-buffers/docs/reference/go-generated

command with which you can install here:
"go install google.golang.org/protobuf/cmd/protoc-gen-go"

Reply:
I added the command and generally referenced the protobuf module. Please see :)